apps/gp2-frontend/src/outputs/export.ts

```diff
@@ -55,8 +83,40 @@ export const outputToCSV = ({
     sorted(workingGroups?.map((workingGroup) => workingGroup.title)) || '',
   projects: sorted(projects?.map((project) => project.title)) || '',
   date: formatDate(new Date(addedDate)),
+  description,
+  shortDescription,
+  relatedEvents: sorted(relatedEvents.map((event) => event.title)) || '',
+  relatedResearch: sorted(relatedOutputs.map((event) => event.title)) || '',
+  gp2Supported: gp2Supported ? 'yes' : 'no',
```

Comment:
I believe `gp2Supported` comes as a string
<img width="999" alt="Screenshot 2024-09-12 at 11 35 18" src="https://github.com/user-attachments/assets/264e6092-3cb3-4bf9-a522-59ee78faafec">

so maybe we could return just

```suggestion
  gp2Supported,
```

because it can also be "Don't Know" I think